demos/full-demo/conjur/secretless_ansible.yml

```diff
@@ -1,11 +1,12 @@
-version: 2
```

Comment:
I wouldn't worry about this change, I remove this folder `demos/full-demo` in #736